.github/ISSUE_TEMPLATE/infrastructure.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Infrastructure Issue
+about: Report an issue with Open Robotics infrastructure
+
+---
+
+- [ ] I have checked the [ROS Status page](https://status.ros.org) and my issue is not listed there.
```

Comment:
I think we should also put a checkbox here for "I have check ROS Answers for my problem and was not able to resolve".

Reply:
ROS Answers is provided as a "contact link" in the config.yaml https://github.com/osrf/infrastructure/pull/1/files#diff-bfe857fb12506e797b48b7815c1d5aec3234a92b11255e0df0c8e9ca7fcf8dd9R3-R5 while we want to do everything we can to push folks first to ROS Answers I also want to avoid patronizing or looping. Do you think the contact link will be sufficient? I didn't include a link to the status page in contact because there's no action users can take on the status page to notify us of an issue.

Reply:
> I didn't include a link to the status page in contact because there's no action users can take on the status page to notify us of an issue.

Yeah, that part makes sense.

> Do you think the contact link will be sufficient?

I think we want to really make sure people are doing all they can before assuming it is an infrastructure issue.  With that in mind, I actually think it is stronger to put a line-item in the issue template rather than having this as a Contact link.  A checkbox that they have to fill out feels like a stronger assurance of "yes, I tried here first" rather than "I just clicked to avoid the contact links when opening the issue".

Reply:
Added checkboxes in bc16490